assets/src/edit-story/components/panels/backgroundOverlay/convertOverlay.js

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { OverlayType } from '../../../utils/backgroundOverlay';
+
+function convertToSolid(currentOverlay, currentType) {
+  switch (currentType) {
+    case OverlayType.NONE:
+      // default color
+      return {
+        color: { r: 0, g: 0, b: 0, a: 0.3 },
+      };
+
+    case OverlayType.LINEAR:
+    case OverlayType.RADIAL: {
+      // converting from any gradient to SOLID
+      // use "opaque'st" stop as primary color, but fix alpha at .3
+      const { stops } = currentOverlay;
+      const stopsByAlpha = stops.sort(
+        (x, y) => (y.color.a ?? 1) - (x.color.a ?? 1)
+      );
+      const {
+        color: { r, g, b },
+      } = stopsByAlpha[0];
+      return {
+        color: { r, g, b, a: 0.3 },
```

Comment:
Why do we fix alpha at .3?
In this case, the following would happen:
1. The user sets solid overlay with opacity 80%.
2. Then the user clicks on "radial".
3. The user changes her mind, clicks "Solid" again -- suddenly the opacity is 30% instead of the original 80%.

Thoughts?